docs/src/testing/gas-etimation.md

```diff
@@ -25,48 +24,9 @@ While using the fuzzing feature additional gas statistics will be displayed:
 > 
 > Estimated gas will always be rounded up to the next integer.
 
-## Calculating Gas
+For further details on how fees are calculated, please refer to fee mechanism 
```

Comment:
Ditching updating this since having this updated is a huge effort (and one that's just not feasible)